types/transform-coordinates/transform-coordinates-tests.ts

```diff
@@ -0,0 +1,9 @@
+import transformation = require('transform-coordinates');
+
+const transform = transformation('EPSG:4326', '3068');
+
+const c1 = transform.forward({x: 13.4105, y: 52.5034});
+console.log(c1.x, c1.y);
```

Comment:
```suggestion
const c1 = transform.forward({x: 13.4105, y: 52.5034});
```
removing this will remove `dom` dependency for DT package